### ISSUE TITLE
Packages to consider 

### ISSUE THREAD

**Issue by OlivierJM** [SparkEdUAB/sparked-mobile#1]
This will be overview of packages that will likely be used in here for different things 

- Typography https://github.com/hectahertz/react-native-typography

**Comment by dennism501** [comment]
@OlivierJM Is that the only package that will be used when creating the components, or are more going to be added?

**Comment by ikayz** [comment]
@OlivierJM cool, I checked them out. So basically they need to be turned into code?

**Comment by OlivierJM** [comment]
Yes they need to be turned into code.

I am still cleaning up project board here https://github.com/SparkEdUAB/sparked-mobile/projects/1 once that is done, we can get started with contributing. 
I would advise you add yourselves to the watch list so that you can know once I start creating issues.

**Comment by ikayz** [comment]
Alright, sounds good 👍🏿

**Comment by OlivierJM** [comment]
@ikayz @dennism501 @yoramgondwe I have setup the project here #6

**Comment by dennism501** [comment]
Hi guys, i am leaving this here for you to check out and see if we can use it for the components, its pretty good https://docs.nativebase.io/

**Comment by OlivierJM** [comment]
Thanks for sharing @dennism501 I think NativeBase is good, am not sure how easy it is to customise to our designs but am open to suggestions.

**Comment by OlivierJM** [comment]
@ikayz @YoramGondwe  you might have to watch the repo to get all updates

**Comment by dennism501** [comment]
> Thanks for sharing @dennism501 I think NativeBase is good, am not sure how easy it is to customise to our designs but am open to suggestions.

I think it can be customized, but then what level of customization would like to do?

**Comment by OlivierJM** [comment]
To the level of the designs https://www.figma.com/file/1llpwoqBGfnmmAqfAtza1K/Spark-Ed---LEARNING-PORTAL?node-id=0%3A1

**Comment by dennism501** [comment]
Yeah, It can be customized to look like the designs. But we have to look at some more before choosing which one to use.

**Comment by OlivierJM** [comment]
> Yeah, It can be customized to look like the designs. But we have to look at some more before choosing which one to use.

can you do that and advise, another example worth taking into consideration is https://rn.mobile.ant.design I only used the web version not sure the react-native version.

**Comment by dennism501** [comment]
Okay, that's fine. Yeah, i have also only used ant on web and not mobile.

**Comment by OlivierJM** [comment]
Alright, you will let me know.

**Comment by dennism501** [comment]
I cloned the repo today and i am getting an error when trying to run it, I am still debugging it. It seems like a gradle version issue

**Comment by OlivierJM** [comment]
oops those I hate with react native 🙈 let me know if you fix it, I only ran on it on IOS emulator.

**Comment by dennism501** [comment]
Yurika!!! Fixed

**Comment by OlivierJM** [comment]
Awesome, thanks

**Comment by OlivierJM** [comment]
I am pinning this issue, so that it will be helpful.

**Comment by dennism501** [comment]
@OlivierJM i also found this one, https://reactnavigation.org/docs/getting-started what are going to use for the routing in the application?

**Comment by OlivierJM** [comment]
> @OlivierJM i also found this one, https://reactnavigation.org/docs/getting-started what are going to use for the routing in the application?

Yeah this one works pretty well.

**Comment by OlivierJM** [comment]
You can actually create an issue to setup routes so that other PR come after it.

**Comment by dennism501** [comment]
Yeah, i think the routes should be set up before everyone starts working the screens, so that we have a format already set for the routes

**Comment by OlivierJM** [comment]
@MwanahamuntuStarr can you help out with this ?

**Comment by OlivierJM** [comment]
Absolutely

**Comment by MwanahamuntuStarr** [comment]
Hey guys, 
#2E8581 is the background color and attached is a zipped folder with the image we are using in SVG format.

[Homepage Illustration.zip](https://github.com/SparkEdUAB/sparked-mobile/files/5433627/Homepage.Illustration.zip)

**Comment by OlivierJM** [comment]
Thanks a lot @MwanahamuntuStarr
